node/resolvers/auth/index.ts

```diff
@@ -14,15 +16,24 @@ const makeRequest = async (ctx, url) => {
 }
 
 export const mutations = {
-  sendEmailVerification: async (_, args, { vtex: ioContext }) => {
-    const { data: { authenticationToken } } = await makeRequest(ioContext, paths.getTemporaryToken())
-    await makeRequest(ioContext, paths.sendEmailVerification(args.email, authenticationToken))
-    return { authToken: authenticationToken }
+  sendEmailVerification: async (_, args, { vtex: ioContext, response }) => {
+    const { data, status } = await makeRequest(ioContext, paths.getTemporaryToken(ioContext.account, ioContext.account))
+    if (!data.authenticationToken) {
+      throw new ResolverError(`ERROR ${data}`, status)
+    }
+    await makeRequest(ioContext, paths.sendEmailVerification(args.email, data.authenticationToken))
+    response.set('Set-Cookie', serialize('temporarySession', data.authenticationToken, { path: '/', }))
+    return data.authenticationToken ? true : false
   },
 
-  signIn: async (_, args, { vtex: ioContext, request: { headers: { cookie } }, response }) => {
-    const { fields: { email, authToken, code } } = args
-    const { data: { authCookie } } = await makeRequest(ioContext, paths.signIn(email, authToken, code))
-    response.set('Set-Cookie', serialize(authCookie.Name, authCookie.Value, { httpOnly: true }))
+  accessKeySignIn: async (_, args, { vtex: ioContext, request: { headers: { cookie } }, response }) => {
+    // FIXME: Get temporarySession in cookies instead of args. @brunojdo - 08/06/2018 
+    const { fields: { email, temporarySession, code } } = args
+    const { data, status } = await makeRequest(ioContext, paths.accessKeySignIn(email, temporarySession, code))
```

Comment:
The `api/vtexid/pub/authentication/accesskey/validate` route does not return a value when the login succeeds. It returns a `200 OK` with no content in the body. (_mental note: change the `200` for a `204`_).

You should get the cookies from the `Set-Cookie` headers in the response.

E.g.:
```http
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 08 Jun 2018 14:07:49 GMT
Content-Type: application/json; charset=utf-8
Connection: close
Vary: Accept-Encoding
Cache-Control: no-store, max-age=0
Set-Cookie: VtexIdclientAutCookie_boticario=eyJhbGciOiJFUzI1NiIsImtpZCI6IkRDMUQ5MUYzOEJEMjY2ODVEMTFCRURBREQ3QkM5NkFBQkM5MjdFQTUiLCJ0eXAiOiJqd3QifQ.eyJzdWIiOiJ0ZXN0LWJlemVycmFAbWFpbGluYXRvci5jb20iLCJhY2NvdW50IjoiYm90aWNhcmlvIiwiYXV0aG9yaXphYmxlcyI6WyJ2cm46aWFtOmJvdGljYXJpbzp1c2Vycy90ZXN0LWJlemVycmFAbWFpbGluYXRvci5jb20iXSwiZXhwIjoxNTI4NTUzMjcwLCJ1c2VySWQiOiJlN2E0MDkwZS01YTJkLTRmYmYtYTljMS1hZjA5MjYzNGM0N2QiLCJpYXQiOjE1Mjg0NjY4MzUsImp0aSI6IjE2NjRjNzA0LWZlYzgtNDBmMy1hNDVhLTc3YTc1YjJmYTllZiIsImlzcyI6InRva2VuLWVtaXR0ZXIifQ.YGLBuYnWZfecAwTPMy0tB6M_bwcMev-ivkJChHnqmaCIROhYWoUKMALLilPxeZ9wYQKVFc21OXhvRQ-1Vq_tiw; expires=Sat, 09 Jun 2018 14:07:48 GMT; domain=; path=/
Set-Cookie: VtexIdclientAutCookie_56fb3a9c-14f2-4fb3-a244-f8bea77a42c1=eyJhbGciOiJFUzI1NiIsImtpZCI6IkRDMUQ5MUYzOEJEMjY2ODVEMTFCRURBREQ3QkM5NkFBQkM5MjdFQTUiLCJ0eXAiOiJqd3QifQ.eyJzdWIiOiJ0ZXN0LWJlemVycmFAbWFpbGluYXRvci5jb20iLCJhY2NvdW50IjoiYm90aWNhcmlvIiwiYXV0aG9yaXphYmxlcyI6WyJ2cm46aWFtOmJvdGljYXJpbzp1c2Vycy90ZXN0LWJlemVycmFAbWFpbGluYXRvci5jb20iXSwiZXhwIjoxNTI4NTUzMjcwLCJ1c2VySWQiOiJlN2E0MDkwZS01YTJkLTRmYmYtYTljMS1hZjA5MjYzNGM0N2QiLCJpYXQiOjE1Mjg0NjY4MzUsImp0aSI6IjE2NjRjNzA0LWZlYzgtNDBmMy1hNDVhLTc3YTc1YjJmYTllZiIsImlzcyI6InRva2VuLWVtaXR0ZXIifQ.YGLBuYnWZfecAwTPMy0tB6M_bwcMev-ivkJChHnqmaCIROhYWoUKMALLilPxeZ9wYQKVFc21OXhvRQ-1Vq_tiw; expires=Sat, 09 Jun 2018 14:07:48 GMT; domain=; path=/
X-VTEX-Janus-Router-Backend-App: vid-v4.78.25
X-Track: stable
X-Powered-by-VTEX-Janus-Edge: v1.44.14
X-CDNIgnore: 1
Content-Encoding: gzip
```

An easy way to test the login behavior is to login to any store. E.g.: <http://www.boticario.com.br/login>.